capstone/src/main/java/com/google/step/clients/ScoreClient.java

```diff
@@ -0,0 +1,160 @@
+package com.google.step.clients;
+
+import com.google.step.data.RestaurantPageViews;
+import com.google.step.data.RestaurantScore;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Collections;
+
+
+/**
+ * Will take care of calculating the score for each restaurant.
+ */
+public class ScoreClient {
+
+    private final float minimum = 10.0f;
```

Comment:
1. Minimum what? This isn't very descriptive.
2. Since this is basically a constant, I'd make it `private static final`
3. In general, just don't use `float`. There's almost never a good reason not to make this a `double`.

Reply:
Changed!